PopulateTools/gobierto
# 1405 - Forbid contributions when container is closed

Closes [#1405](https://github.com/PopulateTools/gobierto/issues/1405)

### What does this PR do?

Prohibits creating contributions, comments and votes inside a contribution container which has already closed or hasn't yet started.

Note that this string: `errors.add(:contribution_container, 'Contributions period has finished')` is not translated because it's not meant to ever be seen by a user.

### How should this be manually tested?

You can go to this contribution container: [http://alcobendas.gobify.net/participacion/p/pregunta/aportaciones](http://alcobendas.gobify.net/participacion/p/pregunta/aportaciones)

If you go to the active container ([http://alcobendas.gobify.net/participacion/p/pregunta/aportaciones/pregunta-al-alcalde](http://alcobendas.gobify.net/participacion/p/pregunta/aportaciones/pregunta-al-alcalde)), everything should behave as normal.

But if you go to the past one ([http://alcobendas.gobify.net/participacion/p/pregunta/aportaciones/contenedor-pasado](http://alcobendas.gobify.net/participacion/p/pregunta/aportaciones/contenedor-pasado)) you shouldn't be able to comment, rate a comment, rate the contribution or flag it. You can also [edit it in the admin](http://alcobendas.gobify.net/admin/participation/processes/15/contribution_containers/23/edit) to start in the future and you shouldn't be able to comment etc. either.

### Does this PR changes any configuration file?

No
